Assets/Shopify/Examples/Scripts/Helpers/ShopifyHelper.cs

```diff
@@ -13,12 +13,12 @@ public class ShopifyNotInitializedException : Exception {
     public static class ShopifyHelper {
         public static bool Initialized;
 
-        public static void Init(string accessToken, string shopDomain) {
+        public static void Init(string accessToken, string shopDomain, string locale = null) {
```

Comment:
isn't the default locale `en`?

Reply:
The default should be `null` because each shop might have different defaults. Without the `Accept-Language` header, the return is the shop default 👍

Reply:
I'm a little surprised that it allows `null` for a `string` 😮